client/src/pages/LandingPage/styles.scss

```diff
@@ -0,0 +1,64 @@
+.landing-page-wrapper {
+	.next-button {
+		width: 100%;
+		height: 2.5em;
+		background-color: #62c44f;
+		font-size: 1.4em;
+		color: white;
+	}
+	.landing-page-container {
+		.game-title {
+			margin-bottom: 0;
+			font-family: Georgia;
+			font-size: 1.8em;
+			font-weight: bold;
+			line-height: 1.2;
+			text-align: left;
+			color: #12008f;
+		}
+
+		.game-type-card-container {
+			width: 100%;
+			height: 87%;
+			.levels-link {
+				height: 100%;
+				display: flex;
+				flex-direction: column;
+				text-decoration: none;
+			}
+		}
+
+		.header-icon {
+			display: flex;
+			justify-content: space-between;
+		}
+	}
+
+	.top-section {
+		display: flex;
```

Comment:
"display: flex" does not work on every browser. You can use sass mixins https://sass-lang.com/guide#topic-6 
put 
  ```
display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
  display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
  display: -ms-flexbox;      /* TWEENER - IE 10 */
  display: -webkit-flex;     /* NEW - Chrome */
  display: flex;             /* NEW, Spec - Opera 12.1, Firefox 20+ */
```
in a mixin